oat-sa/extension-tao-sync
# [TAO-8706] [FE] DEPP - 19EVA2-19EVA6 - Interface for importing a sync file when sync fails

Related to : https://oat-sa.atlassian.net/browse/TAO-8706
 
Display import button to allow user to import synchronization data

#### How to test
 
- install test center and offline instances of `DEPP` extension using composer and seed files from https://oat-sa.atlassian.net/browse/TAO-8681
- create a test center with an assigned sync manager and some delivery
- login into offline instance as a sync manager
- try to run synchronization
- when synchronization fails, click export button
- processing screen should appear
- zip with sync data should be downloaded at the end of the preparation data process
- the initial sync screen should be shown
- click import button and import exported zip file
